evm/verify-token-deployer.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
+
+const axios = require('axios');
+const { Command, Option } = require('commander');
+const { ethers } = require('hardhat');
+const {
+    utils: { defaultAbiCoder, isHexString, keccak256, toUtf8Bytes },
+} = ethers;
+
+const { loadConfig, printError } = require('./utils');
+
+async function processCommand(chain, options) {
+    try {
+        const { signedHash, env, address, message, api } = options;
+        const { txHash } = options;
+        const eventHash = keccak256(toUtf8Bytes('InterchainTokenDeployed(bytes32,address,address,string,string,uint8)'));
+
+        if (!(txHash && txHash.startsWith('0x') && isHexString(txHash) && txHash.length === 66)) {
+            throw new Error('Invalid transaction hash');
+        }
```

Comment:
I think this validation falls under [this](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/evm/utils.js#L283) function in utils, you can also use [this](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/evm/its.js#L224) pattern to validate all params (address, message, api, etc)